asv.conf.json

```diff
@@ -0,0 +1,18 @@
+{
+  "environment_type": "conda",
+  "matrix": {
+    "ipython": [],
+    "matplotlib": [],
+    "nose": [],
+    "numpy": [],
+    "pandas": [],
+    "scipy": []
+  },
+  "project": "scikit-bio",
+  "project_url": "http://scikit-bio.org/",
+  "pythons": [
+    "2.7",
+    "3.4"
```

Comment:
Cool, thanks! If the benchmarks start taking too long to run (once they're fully deployed) we can always drop Python 3.3.